hack/test-go.sh

```diff
@@ -66,9 +78,18 @@ KUBE_TIMEOUT=${KUBE_TIMEOUT:--timeout 60s}
 if [ "${1-}" != "" ]; then
   test_packages="$OS_GO_PACKAGE/$1"
 elif [ -n "$TEST_KUBE" ]; then
-  test_packages=`find_test_dirs; find_upstream_test_dirs`
+  test_packages=`find_test_dirs; special_upstream_test_dirs; find_upstream_test_dirs`
 else
-  test_packages=`find_test_dirs`
+  test_packages=`find_test_dirs; special_upstream_test_dirs`
+fi
+
+if [ -n "$PRINT_PACKAGES" ]; then
```

Comment:
> I added it to avoid hacking around while debugging a problem with a find. I'll take it out if you prefer

yeah, please remove it and then feel free to merge